README.md

```diff
@@ -4,15 +4,60 @@ This is a simple HTTP redirect component written in Go.
 
 This is not a Spin application in itself but a component that can be used in applications to redirect a route.
 
-Example usage:
+## Configuration
 
+The `spin-redirect` component can be configured to address your needs in different scenarios. `spin-redirect` tries to load configuration data from multiple places in the specified order:
+
+1. Spin component configuration
+2. Environment variables
+
+The following table outlines available configuration values:
+
+| Key           | Description                                           | Default Value    |
+|---------------|-------------------------------------------------------|------------------|
+| `destination` | Where should the component redirect to                | *(empty string)* |
+| `statuscode`  | What HTTP status code should be used when redirecting | `302`            |
```

Comment:
Shall we add a mention here (or perhaps elsewhere in these docs; whichever you'd like) of the supported status codes and the fallback/default when an unsupported code is used?

Reply:
Documentation updated: Now all valid status codes are listed